packages/wonder-blocks-popover/src/components/popover.js

```diff
@@ -156,6 +161,7 @@ export default class Popover extends React.Component<Props, State> {
         opened: !!this.props.opened,
         anchorElement: null,
         placement: this.props.placement,
+        contentRef: React.createRef(),
```

Comment:
Then you wouldn't need to to this.

BTW the main reason to do it is to avoid using state for accessing refs, as we might not need to update this internally. React should be able handle this for us (but I might be missing something 😄).

Reply:
Got it! ✅